MLS.Agent.Tools/MLS.Agent.Tools.csproj

```diff
@@ -17,16 +17,17 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Clockwise" Version="1.0.261-beta" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.02" />
-    <PackageReference Include="PocketLogger" Version="0.3.0">
+  <!-- Any new dependency added here must also be included in the Microsoft.DotNet.Try.Markdown, Microsoft.DotNet.Interactive and Microsoft.DotNet.Interactive.Rendering nuspec files -->
+    <PackageReference Include="Clockwise" Version="$(ClockwiseVersion)" />
```

Comment:
I'm really not a fan of this--we're doing something that explicitly makes things weirder and harder in the future (I acknowledge it was not your idea).

Reply:
We should actually look to remove the Clockwise dependency from here.